ngraph/python/CMakeLists.txt

```diff
@@ -73,11 +77,35 @@ file(GLOB_RECURSE SOURCES src/pyngraph/*.cpp)
 pybind11_add_module(_${PROJECT_NAME} MODULE ${SOURCES})
 
 target_include_directories(_${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
-target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::ngraph)
+target_include_directories(_${PROJECT_NAME} PRIVATE "${FRONTEND_INCLUDE_PATH}")
+
+if (NGRAPH_ONNX_IMPORT_ENABLE)
+    target_link_libraries(_${PROJECT_NAME} PRIVATE ngraph::onnx_importer)
```

Comment:
How did it work before? Because I see that python API depends on ONNX importer.

Reply:
This looks not necessary (probably overlooked when cleaned up PoC version)